### ISSUE TITLE
NTR: Subpulmonary stenosis

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#7663]
**Preferred term label:**
Subpulmonary stenosis

**Synonyms (exact)**
Infundibular pulmonary stenosis, Infundibular pulmonic stenosis, subvalvar pulmonary stenosis, subvalvar pulmonic stenosis, subvalvular pulmonary stenosis, subvalvular pulmonic stenosis

**Definition (free text, please give PubMed ID)**
A cardiovascular malformation associated with narrowing of the outflow tract of the ventricle immediately below the pulmonary valve.
PMID:34304616

**Parent term (use hpo.jax.org/app)**
Pulmonic stenosis HP:0001642 (definition/comment to be revised to exclude mention of new child terms)

**Xrefs**
SNOMEDCT_US:448476001, MONDO:0017866

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
